From: Piotr Sokołowski <piotrsok2@gmail.com>
Date: Wed, 13 Sep 2017 05:22:19 +0200
Subject: clang ftbfs

Bug-Debian: https://bugs.debian.org/742972
---
 xball.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/xball.c b/xball.c
index 4df72e4..40d0aca 100644
--- a/xball.c
+++ b/xball.c
@@ -25,8 +25,9 @@
 #include "patchlevel.h"
 
 /*** Start of main program ***/
+int
 main(argc,argv)
-unsigned argc;
+int argc;
 char **argv;
 {
     /* Use the app-defaults file as a set of strings for the fallback */
